spring-cloud-starter-stream-source-http/src/main/resources/META-INF/spring-configuration-metadata-whitelist.properties

```diff
@@ -1,3 +1,4 @@
 configuration-properties.classes=org.springframework.cloud.stream.app.http.source.HttpSourceProperties, \
-  org.springframework.cloud.stream.app.http.source.HttpSourceCorsProperties
+  org.springframework.cloud.stream.app.http.source.HttpSourceCorsProperties, \
+  org.springframework.cloud.stream.app.security.common.SecurityCommonAutoConfigurationProperties
```

Comment:
OK. I agree with this approach and I believe that's how configuration properties are appeared in the README.

Only the problem that we are going to do this kind of change in all the app starters.

Can't we just live without those properties and just mention in the docs that security is disabled by default and can be turned on using particular properties?

I mean that's not what I expected from this PR. 

We should remove all the security related code and if we wish we can include the common module for tests coverage.

Let me know if I am missing something.

Thanks

Reply:
As i mentioned above, at the moment only the http-source uses the security module autoconfiguration.

Reply:
It must not. Since that module is auto-configuration anyway and it is going to appear in each generated app there is no reason to duplicate the logic.
What we need 3is just remove home-made logic in this project since everything is covered by that new module.

We can leave tests here for better coverage and then dependency must go to the `test` scope.

Not sure in your logic to duplicate that module on the `compile` level...

Reply:
+1